lua/neogit/lib/git/stash.lua

```diff
@@ -105,7 +105,7 @@ function M.drop(stash)
 end
 
 function M.list()
-  return cli.stash.args("list").call():trim().stdout
+  return cli.stash.args("list").call_sync():trim().stdout
```

Comment:
I'm sorry but I do not think that solving this by blocking operations is the right call. It will cause the editor to stutter and get small hangups when doing things, especially if you leave the status buffer open.

The `can't yield across C boundary means that the function was called from a callback such as `vim.fn.defer` or similar and not in the callstack of any `a.void`

Reply:
Alright. What do you suggest to prevent the `can't yield across C boundary` issue? For what its worth, there's no discernable stutter/lag of the UI for me.